doc/sphinx/coqdoc.css

```diff
@@ -0,0 +1,338 @@
+/************************************************************************/
+/*         *   The Coq Proof Assistant / The Coq Development Team       */
+/*  v      *   INRIA, CNRS and contributors - Copyright 1999-2018       */
+/* <O___,, *       (see CREDITS file for the list of authors)           */
+/*   \VV/  **************************************************************/
+/*    //   *    This file is distributed under the terms of the         */
+/*         *     GNU Lesser General Public License Version 2.1          */
+/*         *     (see LICENSE file for the text of the license)         */
+/************************************************************************/
+body { padding: 0px 0px;
+       margin: 0px 0px;
+       background-color: white }
+
+#page { display: block;
+        padding: 0px;
+        margin: 0px;
+        padding-bottom: 10px; }
+
+#header { display: block;
+          position: relative;
+          padding: 0;
+          margin: 0;
+          vertical-align: middle;
+          border-bottom-style: solid;
+          border-width: thin }
+
+#header h1 { padding: 0;
+             margin: 0;}
+
+
+/* Contents */
+
+#main{ display: block;
+       padding: 10px;
+       font-family: sans-serif;
+       font-size: 100%;
+       line-height: 100% }
+
+#main  h1 { line-height: 95% } /* allow for multi-line headers */
+
+#main  a.idref:visited {color : #416DFF; text-decoration : none; }
+#main  a.idref:link {color : #416DFF; text-decoration : none; }
+#main  a.idref:hover {text-decoration : none; }
+#main  a.idref:active {text-decoration : none; }
+
+#main  a.modref:visited {color : #416DFF; text-decoration : none; }
+#main  a.modref:link {color : #416DFF; text-decoration : none; }
+#main  a.modref:hover {text-decoration : none; }
+#main  a.modref:active {text-decoration : none; }
+
+#main .keyword { color : #cf1d1d }
+#main { color: black }
+
+.section { background-color: rgb(60%,60%,100%);
+           padding-top: 13px;
+           padding-bottom: 13px;
+           padding-left: 3px;
+           margin-top: 5px;
+           margin-bottom: 5px;
+           font-size : 175% }
+
+h2.section { background-color: rgb(80%,80%,100%);
+             padding-left: 3px;
+             padding-top: 12px;
+             padding-bottom: 10px;
+             font-size : 130% }
+
+h3.section { background-color: rgb(90%,90%,100%);
+             padding-left: 3px;
+             padding-top: 7px;
+             padding-bottom: 7px;
+             font-size : 115% }
+
+h4.section {
+/*
+                   background-color: rgb(80%,80%,80%);
+                   max-width: 20em;
+                   padding-left: 5px;
+                   padding-top: 5px;
+                   padding-bottom: 5px;
+*/
```

Comment:
Could the commented out lines be removed?

Reply:
Definitely